pyTST/core.py

```diff
@@ -44,13 +53,17 @@ def load_data_array(self, signal_array, time_array=None, tstep=1):
 
         tstep : float, optional
             time step used when time_array is not provided
+
+        moving_average : array of float
+            moving average based on the used step size
```

Comment:
moving_average is not a parameter for load_data_array, so this docstring shouldn't be there.